DSESwift/DataPoint.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  DataPoint.swift
+//  SensePlatform
+//
+//  Created by Tatsuya Kaneko on 16/09/15.
+//
+//
+
+import Foundation
+
+class DataPoint{
+    var sensorId = -1
+    private(set) var value = ""
+    var date = NSDate()
+    var synced = false
+
+    init(sensorId: Int, value: String, date: NSDate, synced: Bool) {
+        self.sensorId = sensorId
+        self.value = value
```

Comment:
@kanekin  It can be a little complex for iOS to get the right sensor value type since the getDataPoints is a method of databaseHandler, and the type can only be obtained from the combined sensorId. In Android, type field is stored in both DataPoint class and Sensor class.